src/DependencyInjection/SchedulerBundleExtension.php

```diff
@@ -968,4 +968,9 @@ private function registerDataCollector(ContainerBuilder $container): void
             ])
         ;
     }
+
+    public function getConfiguration(array $config, ContainerBuilder $container)
```

Comment:
Return typehint to add if the interface allows to.

Reply:
BTW, could we add a test for this change?

Reply:
of course

Reply:
is the test enough?

Reply:
Yes